src/auto-evo/simulation/food_source/MarineSnowFoodSource.cs

```diff
@@ -36,7 +39,9 @@ public override float FitnessScore(Species species, SimulationCache simulationCa
             score *= Constants.AUTO_EVO_CHUNK_LEAK_MULTIPLIER;
         }
 
-        score /= energyBalance.FinalBalanceStationary;
+        // I don't think it makes sense here to penalize species with positive ATP balance (with higher balance
+        // reducing the available energy), so that divide by final balance stationary here is gone - hhyyrylainen
+        score *= Constants.AUTO_EVO_MARINE_SNOW_BASE_MULTIPLIER;
```

Comment:
This was supposed to be total consumption, not final balance. There are also other places this mistake is made. This is supposed to be a penalty for wasting energy that doesn't help you get this food source.

Reply:
Shouldn't this be, then `TotalConsumptionStationary`? As you still need to move a bit to find chunks.

Reply:
That makes sense to me.

Reply:
Changed to that.